doc/source/whatsnew/v0.21.1.txt

```diff
@@ -68,7 +68,7 @@ Conversion
 Indexing
 ^^^^^^^^
 
--
+- Bug in ``numpy`` ``array_wrap`` with a pandas ``Series``/``Index`` (:issue:`14042`)
```

Comment:
I believe previous whatsnew convention has been to leave "numpy" unticked

Reply:
make this a more practical note, IOW, what a user would want to know (they dont' care about internals)